src/database/migrations/20191001134213-create-user.js

```diff
@@ -6,10 +6,21 @@ module.exports = {
       username: { type: Sequelize.STRING,  allowNull: false, },
       password: { type: Sequelize.STRING, allowNull: true, },
       email: { type: Sequelize.STRING, allowNull: false, unique: true, },
-      isverified: { type: Sequelize.BOOLEAN, allowNull: true, defaultValue: false },
-      role: { type: Sequelize.STRING, allowNull: true, },
+      isverified: { type: Sequelize.BOOLEAN, allowNull: true, defaultValue: false }, 
       createdAt: { allowNull: false, type: Sequelize.DATE, defaultValue: Sequelize.NOW },
-      updatedAt: { allowNull: false, type: Sequelize.DATE, defaultValue: Sequelize.NOW },
+      updatedAt: { allowNull: false, type: Sequelize.DATE },
+      RoleId: {
+        type: 'integer',
+        onDelete: 'CASCADE',
+        allowNull: true,
+        defaultValue: 5,
+        references: {
+          model: 'Roles',
+          key: 'id',
+          onUpdate: 'CASCADE',
+          onDelete: 'CASCADE'
```

Comment:
in `src/database/models/role.js` you said `onDelete: set null` but here it is `onDelete: 'Cascade'`